frontend/src/credit_transfers/CreditTransferViewContainer.js

```diff
@@ -517,19 +517,27 @@ class CreditTransferViewContainer extends Component {
   }
 
   _modalRecommend (item) {
+    const isBuyOrSell = [CREDIT_TRANSFER_TYPES.buy.id, CREDIT_TRANSFER_TYPES.sell.id].indexOf(item.type.id) >= 0;
```

Comment:
I don't think we need to differentiate between Buy or Sell because the button labels should be the same for both. Is this component even used for Initiative Agreements?

Reply:
Yes, this modal is used for "Transfers with the Submitted Status" as well as for "Initiative Agreements".
Please check out Justin's comment on the [ticket page](https://app.zenhub.com/workspaces/low-carbon-fuel-standard-644c2db3be098a001977ad0b/issues/gh/bcgov/tfrs/2789) for more information.

Reply:
ah gotcha, thanks.

Reply:
You're welcome!
Thanks, Alex, for reviewing my PR.